src/util/config/build.js

```diff
@@ -84,23 +28,6 @@ export const schema = {
 };
 export const config = convict(schema);
 
-// Load environment dependent configuration
-const configPath = path.resolve(
```

Comment:
My vote would be to remove it if HMR is pretty reliable now. Otherwise, the documentation that says not to use ENV vars but rather to use local config for everything except `disable_hmr` would be confusing imo.

For the purpose of this PR though I think it's fine as is, and maybe we can follow up remove.